rebasehelper/application.py

```diff
@@ -804,6 +779,26 @@ def apply_changes(self):
             logger.warning('changes.patch was not applied properly. Please review changes manually.'
                            '\nThe error message is: %s', six.text_type(e))
 
+    def fix_specfile(self, results_dir):
```

Comment:
This shouldn't be called `fix_specfile`, that's misleading. Something like `prepare_next_run` would be more fitting.